client/src/modules/about/About.scss

```diff
@@ -1,15 +1,15 @@
 .about {
   &__intro {
-    margin-top: 5px;
+    margin-top: $s-margin;
     width: 100%;
     background: url('../../assets/images/hero-bg.png') bottom;
     background-size: auto 260px;
     background-repeat: no-repeat;
-    padding: 0 0 100px;
+    padding: 0 0 calc($xxl-padding + $xl-padding + $s-padding);
```

Comment:
could we use `$xxxl-padding` instead of this padding calculation to have our paddings always the same as padding-constants values? Or if we really need padding different from our constants - some scss function which returns proper padding which will be a multiple of four?